scripts/resender.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python3
+
+import click
+import requests
+from ape import project
+from ape.api import AccountAPI
+from ape.cli import ConnectedProviderCommand, account_option
+from ape.contracts import ContractInstance
+from ape.logging import logger
+
+
+def get_release_events(graphql_endpoint: str) -> list[dict]:
+    """
+    Queries GraphQL endpoint to retrieve all new `Release` events
+    on Polygon network
+    """
+
+    gql = (
+        """
+    query ReleasetoResend {
+    releaseds(
+        where: {releaseTxSender: "0x0000000000000000000000000000000000000000", releaseResent: false}
```

Comment:
Out of curiosity how often is the subgraph updated relative to how often this script is run?

i.e. what's the update interval for the subgraph, and what is the interval for running this script?

Reply:
> Out of curiosity how often is the subgraph updated relative to how often this script is run?
> 
> i.e. what's the update interval for the subgraph, and what is the interval for running this script?

With respect to the subgraph, the subgraph is synced and scanning for events in realtime. It may have a little delay, but nothing serious: I just checked this and at this moment is only 8 blocks behind (the frequency of new blocks on Polygon is ~2 sec).